### ISSUE TITLE
Navigation grouping

### ISSUE THREAD

**Issue by adriannelee** [cds-snc/notification-planning#573]
Issue group: Header
Visibility: 3
Evaluation: Partially Supports
Success Criteria: 1.3.1: Info and Relationships
Needs design: Content

### Description
Both mobile navigation menus are contained within the same `<nav>` element labelled “mobile navigation”. The site navigation and “Language and sign in” menus should be separate and properly labelled `<nav>` elements reflecting how they are organized in the desktop presentation.

![]()

**Comment by amazingphilippe** [comment]
Related finding from testing:

https://airtable.com/appWwAmHwDLtpIyko/tbl38n7ofWYBuezFc/viwBYyaQxOttZ0IAw/recAYwlRcb3Q7daMG?blocks=hide

> GC Notify was difficult to navigate for participants. For a user using a screenreader, the labels weren't descriptive enough (e.g. navigation regions, homepage link). For a screen magnification user, the grey banner on the top nav was too light (but other screen magnification users didn't comment on it)

**Comment by amazingphilippe** [comment]
Relevant to #579

**Comment by amazingphilippe** [comment]
Fixed in https://github.com/cds-snc/notification-admin/pull/1301
